### PR TITLE
:bookmark: release 1.1.0

### PR DESCRIPTION
**Pull request by annashamray**

### CODE REVIEW

**Review by joeribekker — CHANGES_REQUESTED** [review]
Add diff to readme oas versions, move 1.1.0 above 1.0.0.
In the changelog move new features to top. Check wording (fixed vs fixes, created vs added)

Will comment on changelog in depth later.

File: CHANGELOG.rst [hunk 1]
```diff
@@ -2,6 +2,52 @@
...
+* Document deployment of the Objects API and the Objecttypes API on single server and Kubernetes (#59)
+* Add AUTHORS file
+* Translate help-texts for `Content-Crs` and `Accept-Crs` headers from Dutch to English in the OAS (#106)
+* Add information about validation to the OAS (#106)
+
+**New features**
+
+* Decouple authentication tokens from users in the admin (#115)
+* Add additional fields for tokens to store extra information (#155)
+* For object types use special ObjectType object instead of urls (#115)
```
Comment (joeribekker) [inline CHANGELOG.rst:34]: Is this a feature? How? Where?

File: CHANGELOG.rst [hunk 2]
```diff
@@ -2,6 +2,52 @@
...
+
+* Improve the Admin UI:
+
+    * Include `uuid` field to the "object" page (#156)
+    * Make `object_type` field immutable (#150)
+    * Add filtering on `object_type` to the "object" page (#157)
```
Comment (joeribekker) [inline CHANGELOG.rst:40]: Move to bottom of this section to keep the other new feature grouped together.

File: CHANGELOG.rst [hunk 3]
```diff
@@ -2,6 +2,52 @@
...
+**Documentation**
+
+* Create a tutorial how to use the Objects API and the Objecttypes API with examples (#61)
+* Document how to configure authentication and authorization for the Objects API and the Objecttypes API (#179)
+* Document deployment of the Objects API and the Objecttypes API on single server and Kubernetes (#59)
+* Add AUTHORS file
+* Translate help-texts for `Content-Crs` and `Accept-Crs` headers from Dutch to English in the OAS (#106)
+* Add information about validation to the OAS (#106)
+
+**New features**
```
Comment (joeribekker) [inline CHANGELOG.rst:30]: Move new features section to top :)

File: CHANGELOG.rst [hunk 4]
```diff
@@ -2,6 +2,52 @@
...
+
+* Decouple authentication tokens from users in the admin (#115)
+* Add additional fields for tokens to store extra information (#155)
```
Comment (joeribekker) [inline CHANGELOG.rst:33]:
```suggestion
* Decoupled authentication tokens from users in the admin (#115)
* Added additional fields for tokens to store extra information (#155)
```

File: CHANGELOG.rst [hunk 5]
```diff
@@ -2,6 +2,52 @@
...
+* For object types use special ObjectType object instead of urls (#115)
+
+* Improve the Admin UI:
```
Comment (joeribekker) [inline CHANGELOG.rst:36]:
```suggestion
* Improved the Admin UI:
```

File: CHANGELOG.rst [hunk 6]
```diff
@@ -2,6 +2,52 @@
...
+
+    * Include `uuid` field to the "object" page (#156)
+    * Make `object_type` field immutable (#150)
+    * Add filtering on `object_type` to the "object" page (#157)
+
+* Adhere the Objecttypes API to API principles API-09, API-18, API-19, API-51 defined in API Design Rules of Nederlandse API Strategie (#46, #174)
+* Support `fields=` query param and display only selected fields in the API response (#174)
+* Add length validation fo url fields (#154)
+
+**Cleanup**
```
Comment (joeribekker) [inline CHANGELOG.rst:46]: Cleanup can be removed or fitted under lib update item:
> * Bumped to newer versions of [...]

File: README.NL.rst [hunk 7]
```diff
@@ -41,6 +41,8 @@ latest          n.v.t.          `ReDoc <https://redocly.github.io/redoc/?url=htt
                                 (`verschillen <https://github.com/maykinmedia/objects-api/compare/1.0.0..master#diff-b9c28fec6c3f3fa5cff870d24601d6ab7027520f3b084cc767aefd258cb8c40a>`_)
 1.0.0           2021-01-13      `ReDoc <https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/maykinmedia/objects-api/1.0.0/src/openapi.yaml>`_,
                                 `Swagger <https://petstore.swagger.io/?url=https://raw.githubusercontent.com/maykinmedia/objects-api/1.0.0/src/openapi.yaml>`_
+1.1.0           2021-04-21      `ReDoc <https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/maykinmedia/objects-api/1.1.0/src/openapi.yaml>`_,
+                                `Swagger <https://petstore.swagger.io/?url=https://raw.githubusercontent.com/maykinmedia/objects-api/1.1.0/src/openapi.yaml>`_
```
Comment (joeribekker) [inline README.NL.rst:45]: Add a diff please (like latest has). Move 1.1.0 above 1.0.0

(same for english version)

File: README.NL.rst [hunk 8]
```diff
@@ -41,6 +41,8 @@ latest          n.v.t.          `ReDoc <https://redocly.github.io/redoc/?url=htt
                                 (`verschillen <https://github.com/maykinmedia/objects-api/compare/1.0.0..master#diff-b9c28fec6c3f3fa5cff870d24601d6ab7027520f3b084cc767aefd258cb8c40a>`_)
```
Comment (joeribekker) [inline README.NL.rst:41]: Change latest diff to compare against 1.1.0

**Review by joeribekker — CHANGES_REQUESTED** [review]

File: CHANGELOG.rst [hunk 9]
```diff
@@ -2,6 +2,52 @@
...
+1.1.0 (2021-04-21)
+------------------
+
+**Bugfixes and QOL**
+
+* Improve performance with database query optimization (#136) :zap:
+* Bumped to newer versions of Django, Jinja2, Pillow, PyYAML including security fixes (#183, #182, #184, #176, #193)
+
+* Fixed a crash when creating an object without a version in the admin (#146)
+* Fixes a crash when opening an objecttype without versions in the admin (maykinmedia/objects-api#144)
```
[the remark this answers is not in the document]
Comment (annashamray) [reply CHANGELOG.rst:14]: Oops, this is the Objecttypes API change. I'll remove this line

File: CHANGELOG.rst (hunk 1 above)
Comment (annashamray) [reply CHANGELOG.rst:33]: It's a part of "Decouple authentication tokens from users", I thought it's worth mentioning in a separate line. I'll remove it

File: CHANGELOG.rst (hunk 2 above)
Comment (annashamray) [reply CHANGELOG.rst]: Ok

File: CHANGELOG.rst (hunk 6 above)
Comment (annashamray) [reply CHANGELOG.rst:45]: Moved to the lib update line

File: README.NL.rst (hunk 7 above)
Comment (annashamray) [reply README.NL.rst:45]: done

File: README.NL.rst (hunk 8 above)
Comment (annashamray) [reply README.NL.rst:41]: done, and I removed the hash from the diff

**Review by joeribekker — APPROVED** [review]

**Pull request merged**